README.md

```diff
@@ -14,8 +14,9 @@ A desktop version update the index.html and style.css
 - FIGMA GIT GITHUB
 - VISUAL STUDIO CODE
 
-## Live
-- https://iptysam.github.io/The_Portfolio/
+## Live 
+
+https://iptysam.github.io/The_Portfolio/
```

Comment:
#### OPTIONAL

- [ ] Kindly use this format for your link `[live demo](https://iptysam.github.io/The_Portfolio/)` so it appears like this  [live demo](https://iptysam.github.io/The_Portfolio/) 👍🏿